### ISSUE TITLE
peerDependencies

### ISSUE THREAD

**Issue by guyellis** [jkphl/gulp-svg-sprite#25]
```
npm WARN peerDependencies The peer dependency vinyl@^0.4.6 included from gulp-svg-sprite will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
```

**Comment by jkphl** [comment]
Hi @guyellis,

thanks for reporting this. Do I understand this right that all I need to "fix" this is list *vinyl* as a regular dependency? (Guess this holds true for **all** peerDependencies in the future?)

Cheers,
Joschi

**Comment by guyellis** [comment]
@jkphl Yes - I believe that's the case.

**Comment by shaneparsons** [comment]
Until there's a fix, what do I have to do from my end to be able to use gulp-svg-sprite?

**Comment by jkphl** [comment]
@shaneparsons Two things:
* The "fix" is [already commited to this repo](https://github.com/jkphl/gulp-svg-sprite/commit/16926ad7c60da42198215b7450871671a16467b5). If you want to, just replace your `package.json` with the most recent one in this repo.
* As far as I understood, all this is just a warning about the future deprecation of a feature in `npm@3`. It should not be a show-stopper of any kind. There is no urgent need to do anything.

However, I will try to push out new releases soon, so you might also just wait and update later today.

**Comment by shaneparsons** [comment]
ah okay good to know, thanks for clearing that up!

**Comment by jkphl** [comment]
Fixed as of [release v1.2.4](https://github.com/jkphl/gulp-svg-sprite/releases/tag/v1.2.4).
